### PR TITLE
[BEAM-5035] Parallel builds are very flaky, disable

### PR DESCRIPTION
**Pull request by apilloud**
Parallel builds have been very flaky for the last few weeks. For example, the post commit has failed in 12 of the last 16 builds due to this issue. We should disable parallelism until it can be fixed.

This extends postcommits to 103 minutes and precommits to 56 minutes. (Given the failure rate, the expected time to a successful precommit is 264 minutes.)

https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/1342/
https://builds.apache.org/job/beam_PreCommit_Java_Phrase/266/

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [X] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [X] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | --- | --- | --- | ---

### CODE REVIEW

**Comment by apilloud** [comment]
run java precommit

**Comment by apilloud** [comment]
run seed job

**Comment by apilloud** [comment]
run java precommit

**Comment by apilloud** [comment]
run java precommit

**Comment by apilloud** [comment]
R: @lukecwik @Ardagan

**Comment by lukecwik** [comment]
R: @alanmyrvold

**Comment by Ardagan** [comment]
FYI: @udim

File: .test-infra/jenkins/PrecommitJobBuilder.groovy [hunk 1]
```diff
@@ -30,7 +30,7 @@ class PrecommitJobBuilder {
   String gradleTask
 
   /** Overall job timeout. */
-  int timeoutMins = 30
```
Comment (udim) [inline .test-infra/jenkins/PrecommitJobBuilder.groovy]: Please undo this change, and only set 120 for Java precommits (with a comment to reset to 30 once BEAM-5035 is fixed).
